Formula/llvm.rb

```diff
@@ -113,22 +74,37 @@ class Llvm < Formula
   depends_on "cmake" => :build
   depends_on :xcode => :build
   depends_on "libffi"
-  depends_on "swig"
 
   def install
+    projects = %w[
+      clang
+      clang-tools-extra
+      lld
+      lldb
+      openmp
+      polly
+    ]
+    runtimes = %w[
+      compiler-rt
+      libcxx
+      libunwind
+    ]
+
+    llvmpath = buildpath/"llvm"
+    unless build.head?
+      llvmpath.install buildpath.children - [buildpath/".brew_home"]
+      (projects + runtimes).each { |p| resource(p).stage(buildpath/p) }
+    end
+
+    # Needed until https://reviews.llvm.org/D63883 lands again.
+    # Use system libcxxabi.
+    rm_r "libcxxabi" if build.head?
+
+    py_ver = "2.7"
```

Comment:
3 isn't an option?

Reply:
I think it is - Linuxbrew uses it. I limited making any actual functional changes (besides fixing polly linking) but I can go ahead and migrate to 3.8 unless there is a particular reason anyone knows not to.

Reply:
If we can use python 3.8, I would prefer that too. Just had no time to migrate it until now.

Reply:
Apart from Python, is there anything else here you would prefer done here that would be useful for Linux?

Reply:
@iMichka I linked the monster patch for linux in a slack PM to Bo

Reply:
I think I'll need to better understand the original problem and why it is Linux specific.

Looking at the patch alone, it doesn't seem obvious at first if there's a better way. You can do things like supplying sysroots but I think that is not what is wanted.